src/schema/v2/infiniteDiscovery/discoverArtworks.ts

```diff
@@ -179,55 +179,96 @@ export const DiscoverArtworks: GraphQLFieldConfig<void, ResolverContext> = {
       user.seenArtworks?.map((node) => node.internalID) || []
     const artworkIdsToFilter = [...likedArtworkIds, ...seenArtworkIds]
 
-    let searchQuery
-
+    let response
     if (likedArtworkIds.length > 0) {
-      searchQuery = gql`
-      {
-        Get {
-          InfiniteDiscoveryArtworks(
-              nearObject: {
-                beacon: "${beacon}",
-                certainty: ${certainty}
-              },
-              limit: ${limit},
-              offset: ${offset},
-          ) {
-            internalID
-            _additional {
-              id
+      const searchQuery = gql`
+        {
+          Get {
+            InfiniteDiscoveryArtworks(
+                nearObject: {
+                  beacon: "${beacon}",
+                  certainty: ${certainty}
+                },
+                limit: ${limit},
+                offset: ${offset},
+            ) {
+              internalID
+              _additional {
+                id
+              }
```

Comment:
**suggestion**: `searchQuery` and `challengeQuery` could work without declaring them in the resolver? I think it would make sense to move them out to make this simpler?